src/js/tabs/adjustments.js

```diff
@@ -286,6 +286,14 @@ adjustments.cleanup = function (callback) {
     if (callback) callback();
 };
 
+$.fn.sortSelect = function() {
+    const op = this.children("option");
+    op.sort(function(a, b) {
+        return a.text > b.text ? 1 : -1;
+    });
+    return this.empty().append(op);
+};
```

Comment:
Maybe we can use the existing function here:

https://github.com/betaflight/betaflight-configurator/blob/bc6307adf74ddd557105d2dd8088267b206e9cbb/src/js/utils/common.js#L90-L103

You call it with sortElement(element);

Reply:
I tried this function first, but It wont work when the text are translated and something else was wrong...

Reply:
Yes it probably only works with single elements.